### PR TITLE
Feature/lp 2935 keyboard onclick alternative

### PR DESCRIPTION
**Pull request by thomaslkp123**

### CODE REVIEW

File: Beis.LearningPlatform.Web/Views/Shared/Components/CmsSocialMedia/Default.cshtml [hunk 1]
```diff
@@ -135,4 +133,14 @@
         return false;
     }
 
+    function setTabOrder() {
+        if (navigator.share) {
+            $('#copy').attr('tabIndex', -1);
```
Comment (JamesChristianson) [inline Beis.LearningPlatform.Web/Views/Shared/Components/CmsSocialMedia/Default.cshtml:138]: Remove jquery please

Comment (JamesChristianson) [inline Beis.LearningPlatform.Web/Views/Shared/Components/CmsSocialMedia/Default.cshtml:138]: remove jq plz

**Review by JamesChristianson — CHANGES_REQUESTED** [review]

**Review by kelechi-i — DISMISSED** [review]
Tested successfully on QA

**Review by emailtokvk — APPROVED** [review]

File: Beis.LearningPlatform.Web/Views/Shared/Components/CmsSocialMedia/Default.cshtml (hunk 1 above)
Comment (thomaslkp123) [reply Beis.LearningPlatform.Web/Views/Shared/Components/CmsSocialMedia/Default.cshtml]: removed

Comment (thomaslkp123) [reply Beis.LearningPlatform.Web/Views/Shared/Components/CmsSocialMedia/Default.cshtml]: removed

**Review by kelechi-i — APPROVED** [review]

**Review by JamesChristianson — APPROVED** [review]

**Pull request merged**